beetsplug/export.py

```diff
@@ -21,7 +21,7 @@
 import codecs
 import json
 import csv
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ElementTree
```

Comment:
I actually kinda liked the `ET` name for `ElementTree`, which is very long, even if a pedantic name style checker thinks it looks like a constant. Maybe we should just call it `et` or something, which at least looks like a module name.

If we stick with this, though, the more standard way to write it would be:

```py
from xml.etree import ElementTree
```

Reply:
Flake8 also made this suggestion. Not sure if there is an explicit ignore comment we can make, but fine doing so if there is.